src/applications/mhv-landing-page/utilities/data/index.js

```diff
@@ -3,6 +3,8 @@ import { mhvUrl } from '@department-of-veterans-affairs/platform-site-wide/utili
 // Links to MHV subdomain need to use `mhvUrl`. Va.gov links can just be paths
 import { HEALTH_TOOL_HEADINGS, HEALTH_TOOL_LINKS } from '../../constants';
 
+const isLinkData = x => x?.href !== undefined && x?.text !== undefined;
```

Comment:
I am wondering if this could be better as just a test that breaks if the data is not populated correctly. This way the build is stopped. When would having non-links here really happen?

Reply:
I created this near the conditional expressions in `myVaHealthBenefitsLinks` (line 73 on this branch) and `moreResourcesLinks` (line 96) that cause the breakage. 

I think this could also be done by having the HubLinks component not attempt to render invalid input, but feel like it might be better to do when the data is instantiated rather than within the React render loop. I would be open to alternatives as well

Reply:
To clarify, I saw this when running the shared mhv platform mocks with `vaPatient: true` added to the user mock. I was adding `vaPatient: true` to the shared mhv platform mocks because the landing page links won't appear when that value isn't true (and it should be true for users with one or more facilities)